tests/services/synapse/test_ops.py

```diff
@@ -34,3 +68,155 @@ def test_monitor_evaluation_queue_returns_true_when_there_are_new_submissions(
+
+        # Assertions
+        syn_mock.tableQuery.assert_called_once_with(
+            f"select * from {submission_view} limit 1"
+        )
+
+
+def test_get_submissions(mocker: pytest.fixture, mocked_ops: MagicMock) -> None:
+    """
+    Tests that the ``get_submissions`` method in ``SynapseOps``
+    returns a list of submission IDs.
+
+    Arguments:
+        mocker: A mocker object.
+        mocked_ops: A mocked instance of ``SynapseOps``.
+
+    """
+    # Input values
+    submission_view = "syn111"
+    submission_status = "RECEIVED"
+
+    input_dict = {
+        "id": ["submission_1", "submission_2"],
+        "status": ["RECEIVED", "RECEIVED"],
+    }
+
+    # Mocking connection to the Synapse API
+    syn_mock = mocked_ops.client
+
+    # Mocking the table query results
+    table_mock = MagicMock()
+    table_mock.asDataFrame.return_value = pd.DataFrame(input_dict)
+
+    # Mock the ``trigger_indexing`` call in SynapseOps() and the tableQuery call
+    # in ``SynapseOps().get_submissions``
+    with mocker.patch.object(mocked_ops, "trigger_indexing", return_value=None):
+        with mocker.patch.object(syn_mock, "tableQuery", return_value=table_mock):
+            # Calling the function to be tested
+            result = mocked_ops.get_submissions(submission_view, submission_status)
+
+            # Assertions
+            syn_mock.tableQuery.assert_called_once_with(
+                f"select * from {submission_view} where status = '{submission_status}'"
+            )
+            table_mock.asDataFrame.assert_called()
+            assert result == input_dict["id"]
+
+
+def test_update_submissions_status_with_input_list(
+    mocker: pytest.fixture, mocked_ops: MagicMock
+) -> None:
+    """
+    Tests that the ``update_submissions_status`` method in ``SynapseOps``
+    updates the status of one or more submissions in Synapse.
+
+    Arguments:
+        mocker: A mocker object.
+        mocked_ops: A mocked instance of ``SynapseOps``.
+    """
+    # Mocking the ``update_submission_status`` call in ``SynapseOps``
+    mock_update_status = mocker.patch.object(mocked_ops, "update_submissions_status")
+
+    # Calling the function to be tested with an input list of strings
+    mocked_ops.update_submissions_status(["submission_1", "submission_2"], "SCORED")
+
+    # Assertions
+    mock_update_status.assert_called_once_with(
+        ["submission_1", "submission_2"], "SCORED"
+    )
+
+
+def test_update_submissions_status_with_input_string(
+    mocker: pytest.fixture, mocked_ops: MagicMock
+) -> None:
+    """
+    Tests that the ``update_submissions_status`` method in ``SynapseOps``
+    updates the status of one or more submissions in Synapse.
+
+    Arguments:
+        mocker: A mocker object.
+        mocked_ops: A mocked instance of ``SynapseOps``.
+
+    """
+    # Mocking the ``update_submission_status`` call in ``SynapseOps``
+    mock_update_status = mocker.patch.object(mocked_ops, "update_submissions_status")
```

Comment:
You shouldn't be mocking the function you are testing, you should be mocking dependent functions within it